rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -791,6 +791,14 @@ async fn run_controller(
         .await
         .context("Failed to read / create device ID")?;
 
+    if device_id.generated {
```

Comment:
I'm probably missing something, but wouldn't this be `if !device_id.generated`?

Reply:
I guess it's ambiguous between "generated ever" and "generated just now"

If I change the variable name to `generated_just_now` or maybe `first_time` would that clear it up?

Reply:
Yep! I had the same thought.

Reply:
Yep! I had the same thought.

Reply:
Changed it to `is_first_time`